pkgs/development/interpreters/ruby/build-ruby-gem/gem-post-build.rb

```diff
@@ -64,8 +64,7 @@
 # this file is here to facilitate running it.
 #
 
-gem_path = ENV["GEM_PATH"]
-ENV["GEM_PATH"] = "\#{gem_path}\#{":" unless gem_path.nil? || gem_path.empty?}#{(gem_path+[gem_home]).join(":")}"
+Gem.use_paths "#{gem_home}", #{gem_path.to_s}
```

Comment:
isn't it `gem_path.inspect` ? If gem_path is nil then we would get `Gem.use_paths "/path/to/gem_home",`